scrape/helpers_test.go

```diff
@@ -253,7 +253,7 @@ func (a *collectResultAppender) String() string {
 	return sb.String()
 }
 
-// protoMarshalDelimited marshals a MetricFamily into a delimited
+// protoMarshalDelimited marshals a MetricDescriptor into a delimited
```

Comment:
If I understood you correctly, you're not using the term "MetricDescriptor" anymore? Should this be replaced with something else?

Reply:
yes, thanks - it might accidental rename, wil fix